openpmd_viewer/addons/pic/lpa_diagnostics.py

```diff
@@ -445,8 +445,8 @@ def get_current( self, t=None, iteration=None, species=None, select=None,
         # Return the current and bin centers
         return(current, info)
 
-    def get_laser_envelope( self, t=None, iteration=None, pol=None, m='all',
-                            theta=0, slice_across=None,
+    def get_laser_envelope( self, t=None, iteration=None, pol=None, laser_coord='z',
```

Comment:
Would you mind changing the name of this variable to `laser_propagation`, as it is slightly more self-explanatory.
```suggestion
    def get_laser_envelope( self, t=None, iteration=None, pol=None, laser_propagation='z',
```